gnosis/cowswap
# Affiliate status messages

# Summary
Closes #1343 

Change status messages on Affiliate program.
![image](https://user-images.githubusercontent.com/622217/138165089-00dbf627-8f2a-4703-a22d-79c179867786.png)

  # To Test
Go to [Profile page](https://pr1675--gpswapui.review.gnosisdev.com/#/profile)
Cases: (**I Need help here to complete the steps to reproduce each case**)
1. Not connected: (complete steps). You should see the message: "Affiliate program: Please connect your wallet to participate."
2. Own link: copy your referral link and paste it in the browser. You should see the message: "Affiliate program: Your affiliate code works! Any new user following this link would credit you their trading volume."
3. Already traded: (complete steps). You should see the message: "Invalid affiliate code: The currently connected wallet has traded before or is already part of the affiliate program."
4. Active: (complete steps). You should see the message: "Valid affiliate code: You can now do your first trade to join the program."
5. Unsupported network. Change the network (different to Mainnet). You should see the message: "Affiliate program: Only Mainnet is supported. Please change the network to participate."
6. Affiliate program error: Switch off the Internet, and follow a referral link. You should see the message: "There was an error validating existing trades. Please try again."
7. Affiliate program error: (complete steps). You should see the message: "There was an error while uploading the referral document to IPFS. Please try again."
